packages/plugins/rubygem-google-protobuf/rubygem-google-protobuf.spec

```diff
@@ -0,0 +1,83 @@
+# template: default
+%global gem_name google-protobuf
+%global gem_require_name %{gem_name}
+
+Name: rubygem-%{gem_name}
+Version: 3.21.6
+Release: 1%{?dist}
+Epoch: 1
```

Comment:
I may have missed this, but why is there an epoch?

Reply:
That's a great question, I cannot find a reason for having one as I do not see any competing RPMs that exist and we have not released one ourselves. I think then you can just drop this line and revert the changelog @stejskalleos